### PR TITLE
Link update & minor codestyle improvements

### LINKED ISSUE #9081: "throttling is enhanced" link is dead
The link in the Delivery Optimization section to information about enterprise throttling (https://docs.microsoft.com/en-us/windows-insider/at-work-pro/wip-4-biz-whats-new#new-download-throttling-options-for-delivery-optimization-build-18917) goes to a 404 not found page. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: d602e13e-c1af-958b-8574-a020c49a4b89
* Version Independent ID: f77197b5-cabd-25e7-473a-f4aa663ff6e9
* Content: [What's new in Windows 10 deployment - Windows Deployment](https://docs.microsoft.com/en-us/windows/deployment/deploy-whats-new#delivery-optimization)
* Content Source: [windows/deployment/deploy-whats-new.md](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/public/windows/deployment/deploy-whats-new.md)
* Product: **w10**
* Technology: **windows**
* GitHub Login: @greg-lindsay
* Microsoft Alias: **greglin**

### PR DESCRIPTION
**Pull request by illfated**
**Description:**

From issue ticket #9081 (**"throttling is enhanced" link is dead**):
> The link in the Delivery Optimization section to information about enterprise throttling (https://docs.microsoft.com/en-us/windows-insider/at-work-pro/wip-4-biz-whats-new#new-download-throttling-options-for-delivery-optimization-build-18917) goes to a 404 not found page.

Thanks to @dongearhart for noticing and reporting this issue.

**Changes proposed:**

- Update link to the archived content "New download throttling options for Delivery Optimization (Build 18917)"

**Codestyle & whitespace:**

- Remove redundant end-of-line blanks (17 occurrences)
- Normalize spacing after the dash in bullet point lists, from 3 spaces to 1 (1 occurrence)
- Add missing colon in "Applies to:"

**Ticket closure or reference:**

Closes #9081

### CODE REVIEW

**Review by illfated — COMMENTED** [review]
- **Regular source view:** https://github.com/MicrosoftDocs/windows-itpro-docs/pull/9092/files
- **Hide whitespace changes:** (shortens the page review length quite a lot) https://github.com/MicrosoftDocs/windows-itpro-docs/pull/9092/files?diff=unified&w=1

**Comment by illfated** [comment]
@JohanFreelancer9 : May I request a copy review, please (using the expanded, regular view).

**Comment by illfated** [comment]
#sign-off

File: windows/deployment/deploy-whats-new.md [hunk 1]
```diff
@@ -61,16 +61,16 @@ Windows PowerShell cmdlets for Delivery Optimization have been improved:
...
+- Enterprise network [throttling is enhanced](https://docs.microsoft.com/windows-insider/archive/new-for-business#new-download-throttling-options-for-delivery-optimization-build-18917) to optimize foreground vs. background throttling.
 - Automatic cloud-based congestion detection is available for PCs with cloud service support.
-- Improved Peer Efficiency for enterprises and educational institutions with complex networks is enabled with of [new policies](https://docs.microsoft.com/windows/client-management/mdm/policy-csp-deliveryoptimization). This now supports Microsoft 365 Apps for enterprise updates, and Intune content, with Microsoft Endpoint Manager content coming soon! 
+- Improved Peer Efficiency for enterprises and educational institutions with complex networks is enabled with of [new policies](https://docs.microsoft.com/windows/client-management/mdm/policy-csp-deliveryoptimization). This now supports Microsoft 365 Apps for enterprise updates, and Intune content, with Microsoft Endpoint Manager content coming soon!
```
Comment (JohanFreelancer9) [inline windows/deployment/deploy-whats-new.md:66]:
```suggestion
- Improved peer efficiency for enterprises and educational institutions with complex networks is enabled with [new policies](https://docs.microsoft.com/windows/client-management/mdm/policy-csp-deliveryoptimization). This now supports Microsoft 365 Apps for enterprise updates and Intune content, with Microsoft Endpoint Manager content coming soon!
```

File: windows/deployment/deploy-whats-new.md [hunk 2]
```diff
@@ -61,16 +61,16 @@ Windows PowerShell cmdlets for Delivery Optimization have been improved:
...
 
 The following Delivery Optimization policies are removed in the Windows 10, version 2004 release:
 
 - Percentage of Maximum Download Bandwidth (DOPercentageMaxDownloadBandwidth)
   - Reason: Replaced with separate policies for foreground and background
 - Max Upload Bandwidth (DOMaxUploadBandwidth)
-  - Reason: impacts uploads to internet peers only, which isn't used in Enterprises. 
+  - Reason: impacts uploads to internet peers only, which isn't used in Enterprises.
```
Comment (JohanFreelancer9) [inline windows/deployment/deploy-whats-new.md:73]:
```suggestion
  - Reason: impacts uploads to internet peers only, which isn't used in enterprises.
```

File: windows/deployment/deploy-whats-new.md [hunk 3]
```diff
@@ -104,7 +104,7 @@ For more information, see [Windows 10 Enterprise E3 in CSP](windows-10-enterpris
 
 ### Windows Autopilot
 
-[Windows Autopilot](https://docs.microsoft.com/windows/deployment/windows-autopilot/windows-autopilot) streamlines and automates the process of setting up and configuring new devices, with minimal interaction required from the end user. You can also use Windows Autopilot to reset, repurpose and recover devices. 
+[Windows Autopilot](https://docs.microsoft.com/windows/deployment/windows-autopilot/windows-autopilot) streamlines and automates the process of setting up and configuring new devices, with minimal interaction required from the end user. You can also use Windows Autopilot to reset, repurpose and recover devices.
```
Comment (JohanFreelancer9) [inline windows/deployment/deploy-whats-new.md:107]:
```suggestion
[Windows Autopilot](https://docs.microsoft.com/windows/deployment/windows-autopilot/windows-autopilot) streamlines and automates the process of setting up and configuring new devices, with minimal interaction required from the end user. You can also use Windows Autopilot to reset, repurpose, and recover devices.
```

File: windows/deployment/deploy-whats-new.md [hunk 4]
```diff
@@ -138,11 +138,11 @@ During the upgrade process, Windows Setup will extract all its sources files to
...
 
-Upgrade Readiness helps you ensure that applications and drivers are ready for a Windows 10 upgrade. The solution provides up-to-date application and driver inventory, information about known issues, troubleshooting guidance, and per-device readiness and tracking details. 
+Upgrade Readiness helps you ensure that applications and drivers are ready for a Windows 10 upgrade. The solution provides up-to-date application and driver inventory, information about known issues, troubleshooting guidance, and per-device readiness and tracking details.
 
-The development of Upgrade Readiness has been heavily influenced by input from the community the development of new features is ongoing. To begin using Upgrade Readiness, add it to an existing Operation Management Suite (OMS) workspace or sign up for a new OMS workspace with the Upgrade Readiness solution enabled.  
+The development of Upgrade Readiness has been heavily influenced by input from the community the development of new features is ongoing. To begin using Upgrade Readiness, add it to an existing Operation Management Suite (OMS) workspace or sign up for a new OMS workspace with the Upgrade Readiness solution enabled.
```
Comment (JohanFreelancer9) [inline windows/deployment/deploy-whats-new.md:145]:
```suggestion
The development of Upgrade Readiness has been heavily influenced by input from the community; the development of new features is ongoing. To begin using Upgrade Readiness, add it to an existing Operation Management Suite (OMS) workspace or sign up for a new OMS workspace with the Upgrade Readiness solution enabled.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - a few minor additional edits, please.

File: windows/deployment/deploy-whats-new.md (hunk 1 above)
Comment (illfated) [reply windows/deployment/deploy-whats-new.md:66]: Very good, thank you for noticing those improvement opportunities. 👍🏻

File: windows/deployment/deploy-whats-new.md (hunk 2 above)
Comment (illfated) [reply windows/deployment/deploy-whats-new.md:73]: I half disagree with this one. Lowercase "enterprise" is OK, I agree with that one.
But to me it looks like the word "impacts" is a verb here and it refers back to the singular "Max Upload Bandwidth (DOMaxUploadBandwidth)" policy name. Have I misunderstood the sentence?

File: windows/deployment/deploy-whats-new.md (hunk 3 above)
Comment (illfated) [reply windows/deployment/deploy-whats-new.md:107]: Oxford comma. Nice one. Thanks.

File: windows/deployment/deploy-whats-new.md (hunk 4 above)
Comment (illfated) [reply windows/deployment/deploy-whats-new.md:145]: Nice, it is rare to see actual correct use of semicolon these days, unless I browse technical literature.

**Comment by illfated** [comment]
Very good, thank you for your review and improvement suggestions, those are quite useful and good.

**Comment by illfated** [comment]
Very good job. Thank you for your effort & time.

**Comment by illfated** [comment]
#sign-off

**Comment by illfated** [comment]
To the MS Docs team member who performs the final review before merge: Please note that "Hide whitespace changes" is very useful to shorten the review page length:
- https://github.com/MicrosoftDocs/windows-itpro-docs/pull/9092/files?diff=unified&w=1

File: windows/deployment/deploy-whats-new.md (hunk 2 above)
Comment (JohanFreelancer9) [reply windows/deployment/deploy-whats-new.md]: You are right, thank you.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by nam31** [comment]
@Kateyanne, PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**